docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Future Release
 ==============
     * Enhancements
     * Fixes
+        * Update ``Trend`` primitive to work with ``IntegerNullable`` inputs (:pr:`2204``)
```

Comment:
Also fixes RollingTrend since it uses the same function.

Reply:
Good call - I updated to reflect that.